features/step_definitions/run_steps.rb

```diff
@@ -48,7 +48,7 @@
 
 
 
-Then(/^I get the error:$/) do |error_message|
+Then(/^(?:Git Town|it) prints the error:$/) do |error_message|
```

Comment:
I think the ability to use `Git Town` or `it` should be removed in favor of always using it. Same for the rest of this file.

Reply:
Okay, done.